boards/nxp/rd_rw612_bga/Kconfig

```diff
@@ -0,0 +1,5 @@
+# Copyright 2024 NXP
```

Comment:
2025. Also there is an overlap with this PR: https://github.com/zephyrproject-rtos/zephyr/pull/87594

Reply:
Thank you for making me aware of this PR I will stay in sync, also I will update the copyrights.